src/Rings/groebner.jl

```diff
@@ -304,6 +307,30 @@ function groebner_basis(I::MPolyIdeal; ordering::MonomialOrdering = default_orde
     return standard_basis(I, ordering=ordering, complete_reduction=complete_reduction, algorithm=algorithm)
 end
 
+@doc raw"""
+    standard_basis_highest_corner(I::MPolyIdeal; ordering::MonomialOrdering = default_ordering(base_ring(I))) 
+
+Return a standard basis of `I` with respect to `ordering`. `ordering` needs to be local, the coefficient ring needs to be `QQ`.
+The algorithm first computes a standard basis over a finite field in order to get a highest corner fast. Then this highest corner
```

Comment:
This is acutally only an upper bound for the highest corner w.r.t the monomial ordering, but this suffices to speed up the computation anyway. Maybe this should be stated in that way.